tools/test-static-code

```diff
@@ -49,7 +49,7 @@ fi
 # wrongly marked translatable strings
 #
 
-if out=$(find src/ pkg/ -name '*.js' -o -name '*.jsx' | xargs grep "_('"); then
+if out=$(find src/ pkg/ -name '*.js' -o -name '*.jsx' | xargs grep "_(['\`]"); then
```

Comment:
sorry, but it was typo, and \` is not a problem. Although it is also not bad to keep it in mind.

However there are different problems. `cockpit.gettext` can have two arguemnts - where first is context and second one is string. In such case this matching does not work, as seen for example [here](https://github.com/cockpit-project/cockpit/blob/master/pkg/lib/journal.js#L301).

Also another big problem is that a lot of `cockpit.ngettext` have arguments with single quotes (sometimes all arguments, sometimes only a few of them). (see for yourself `git grep ngettext pkg`)